src/components/SignInPage.js

```diff
@@ -0,0 +1,25 @@
+import "./SignInPage.css";
+
+export default function SignInPage() {
+  return (
+    <div className="app-grid">
```

Comment:
Some CSS conventions info. I link a comment I already wrote to Rikki
https://github.com/FriederikeM/routinely/pull/9#discussion_r620908613